### DISCUSSION TITLE
[Camera] enabling parallel projection in global/custom theme

### DISCUSSION THREAD

**Discussion by MarkusZimmerDLR** [pyvista/pyvista#5612] [Q&A] [answered]
Dear pyvista folks,

I am trying to activate parallel projection for a complete project by setting it in the global or a custom theme.

However, setting doesn't do anything

```python
>>> from pyvista import themes
>>> import pyvista as pv
>>> t = themes.DocumentTheme()
>>> t.camera['parallel_projection']=True
>>> t.camera['_parallel_projection']=True
>>> p = pv.Plotter(theme=t)
>>> p.camera.parallel_projection
False
```

A quick look into the source code of the Camera class reveals:
```python
class Camera(_vtk.vtkCamera):
    def __init__(self, renderer=None):
        """Initialize a new camera descriptor."""
        self._parallel_projection = False
        self._elevation = 0.0
        self._azimuth = 0.0
        self._is_set = False
```
And I cannot find a reference to the theme anywhere.

Am I missing something? The camera key actually exists in some themes.

Cheers
Markus

**Comment by MatthewFlamm** [comment, marked as answer]
The `camera` attribute of a Theme only sets the position. See [Theme.camera docs](https://docs.pyvista.org/version/stable/api/plotting/_autosummary/pyvista.plotting.themes.theme.camera).  Adding a `parallel_projection` option on a theme sounds like a good feature request though.

**Reply by MarkusZimmerDLR** [reply]
Thanks!
